src/components/FilterBox/FilterModuleBig.css

```diff
@@ -1,59 +0,0 @@
-.Module {
-    position: absolute;
-    top: -200px;
-    left: 115%;
-    width: 300px;
```

Comment:
Besser mit Prozentzahlen arbeiten.

Reply:
Bzw. benutzt doch besser ein CSS-Grid Layout für alle Layoutangaben.

Reply:
Git markiert gelöschten Code in Commits rot. FilterBox existiert schon seit ein paar Tagen nicht mehr.